dpsim/src/Simulation.cpp

```diff
@@ -177,6 +171,7 @@ void Simulation::sync() {
 	mLog->info("Start synchronization with remotes on {} interfaces", numOfSyncInterfaces);
 
 	for (auto ifm : mInterfaces) {
+		//TODO: Change or document this synchronization behavior
 		if(ifm.syncStart) {
```

Comment:
This is used to synchronize two real-time simulators in the beginning of a co-simulation. During the co-simulation, the two simulators may rely only on their own clocks and no synchronization mechanism to minimize the impact of communication on the real-time performance / variance of the timestep.